packages/twenty-server/src/workspace/messaging/services/messaging-utils.service.ts

```diff
@@ -11,12 +11,14 @@ import {
 import { MessageQuery } from 'src/workspace/messaging/types/message-or-thread-query';
 import { MessageChannelMessageAssociationService } from 'src/workspace/messaging/message-channel-message-association/message-channel-message-association.service';
 import { MessageService } from 'src/workspace/messaging/message/message.service';
+import { MessageThreadService } from 'src/workspace/messaging/message-thread/message-thread.service';
 
 @Injectable()
 export class MessagingUtilsService {
```

Comment:
Why do we need a messaging utils service AND a service for each entity?

Reply:
This is legacy, I'm slowly moving it everything from this class to dedicated services

Reply:
In utils, we put functions that are used by the different services